app/Rules/Recaptcha.php

```diff
@@ -32,4 +32,15 @@ public function message()
     {
         return 'The recaptcha verification failed. Try again.';
     }
+
+    /**
+     * Is Recaptcha in test mode?
+     * @method isInTestMode
+     *
+     * @return   bool
+     */
+    public static function isInTestMode()
+    {
+        return config('services.recaptcha.key') == '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI';
```

Comment:
Where are we getting this key from? Is it random? Is it provided by Google?

Reply:
It is provided by google as you can find out here: https://developers.google.com/recaptcha/docs/faq

See also: #8

Reply:
It feels kinda gross to me to have the value hard-coded like this, it may be more robust to actually submit an arbitrary test value to Recaptcha to see if it passes.

Reply:
Fixed.